### DISCUSSION TITLE
Shared Songs Need Volume Slider!

### DISCUSSION THREAD

**Discussion by zananashi** [navidrome/navidrome#3137] [Ideas]
Currently it does not have any way to control volume, which is really inconvenient! Would really help if this was included in the future

**Comment by deluan** [comment]
Yeah, I agree, but this will have to wait for the work on the new UI to be completed.

**Comment by ape364** [comment]
bump :)

**Comment by nekiak** [comment]
bump !

**Comment by UuBroot** [comment]
I am willing to work on this, but getting the shares working in the dev enviroment is kind of tricky :/ 
I'll see what i can do, but since i have never contributed to navidrome before this might take some time for this honestly pretty simple thing (at least it seams simple from the code i've read) to implement.
